lib/lte_link_control/Kconfig

```diff
@@ -349,6 +349,57 @@ config LTE_LC_WORKQUEUE_STACK_SIZE
 	default 1280 if LOG_MODE_IMMEDIATE
 	default 1024
 
+choice LTE_UICCPOWERSAVE_MODE
```

Comment:
I'll confirm from the modem team whether the configuration is stored into NVM or not. At least it's not mentioned in the AT command specification. If it would be stored, one option would be that "default" does nothing, i.e. uses the configuration from modem NVM.

Reply:
I confirmed from the modem team, that `%UICCPOWERSAVE` setting is not saved into NVM, so it should be always disabled/default/0 when the modem starts.